assignment1/lib/models/raw_items.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:assignment1/models/item_details.dart';
+//import 'package:assignment1/models/print_class_for_raw_and_manufactured_items.dart';
+
+class RawItems extends ItemDetails {
+  String? itemName, itemType;
+  double? itemPrice, itemQuantity;
+  RawItems(this.itemName, this.itemPrice, this.itemQuantity, this.itemType);
+
+  @override
+  double calculateTax() {
+    return 0.125 * itemPrice!;
```

Comment:
Define 0.125 by converting it into a constant and giving it a name that represents its use